README.md

```diff
@@ -60,27 +60,54 @@ pip install --user mal
 
 ### Manual Installation
 
-Clone this project and run inside it:
+Clone this project and inside it run:
 
 ```
 make install
```

Comment:
You miss that part that I said before? Please, remove that I write this instead: `pip install --user .`